hi-ml-histopathology/src/histopathology/utils/output_utils.py

```diff
@@ -332,7 +333,10 @@ def save_validation_outputs(self, epoch_results: EpochResultsType, metrics_dict:
             self.tiles_selector.gather_selected_tiles_across_devices()
 
         # Only global rank-0 process should actually render and save the outputs
```

Comment:
Could you update this comment to explain the new scenario?

Reply:
Sure!